media/js/firefox/share-page.es6.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+(function () {
```

Comment:
**nitpick:** No need for an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) here, since Webpack will add one automatically around the code anyways.